public/index.php

```diff
@@ -120,6 +120,7 @@ function fatal($msg)
 //===============================================
 require( SYS_PATH.'kissmvc.php' );
 require( APP_PATH.'helpers/site_helper'.EXT );
+require( APP_PATH.'helpers/ip_blocker.php' );
```

Comment:
I don't want the ip_blocker loaded for everyone, please load dynamically via autoloader in the auth controller.